gapic/packaging/api_defaults.yaml

```diff
@@ -21,19 +21,24 @@ homepage: https://github.com/googleapis/googleapis
 license: Apache-2.0
 release_level:
   python: alpha
+  java: alpha
 generated_package_version:
-  # TODO: Python must go to 1.0.20 or 1.1.0 when we GA, due to mistakenly
-  #       publishing some 1.0.x versions to pypi already. They are now
-  #       deleted but files still exist on the pypi servers and can prevent
-  #       us from using the version number again.
-  python:
-    lower: '0.15.4'
-    upper: '0.16dev'
-  nodejs:
-    lower: '0.7.1'
-  php:
-    lower: '0.1.0'
-  ruby:
-    lower: '0.6.8'
-  java:
-    lower: '0.1.15'
+  non_ga:
+    # TODO: Python must go to 1.0.20 or 1.1.0 when we GA, due to mistakenly
+    #       publishing some 1.0.x versions to pypi already. They are now
+    #       deleted but files still exist on the pypi servers and can prevent
+    #       us from using the version number again.
+    python:
+      lower: '0.15.4'
+      upper: '0.16dev'
+    nodejs:
+      lower: '0.7.1'
+    php:
+      lower: '0.1.0'
+    ruby:
+      lower: '0.6.8'
+    java:
+      lower: '0.1.15'
+  ga:
```

Comment:
Done as you suggested. Updated toolkit and artman as well.

I considered using this approach (generated_ga_package_version) at first but I decided to go with the other way just to avoid adding another top-level field.